safe-global/web-core
# fix: Always display pinned apps on top in dashboard

## What it solves

Resolves #831 

## How this PR fixes it

- Adjusts ranking function to exclude pinned apps
- Always shows pinned apps before ranked and random apps

## How to test it

1. Open the Safe
2. Navigate to the dashboard
3. It is possible to pin safe apps
4. Pinned apps always appear in the first position
5. The list still only shows 5 apps
6. Random apps at the end of the list are not reordering anymore when pinning/unpinning apps
7. Reloading the page keeps the pinned state of the apps

## Screenshots
<img width="1278" alt="Screenshot 2022-10-07 at 12 18 05" src="https://user-images.githubusercontent.com/5880855/194531459-fff3d594-ebcb-4263-a9b6-8ae7a8cc5d0c.png">
